### ISSUE TITLE
PsiViewer ToolWindow icon is 13x13, but warned it isn't

### ISSUE THREAD

**Issue by rgoldberg** [JetBrains/psiviewer#31]
Using PsiViewer, I saw the following in the IntelliJ logs:

> WARN - openapi.wm.impl.ToolWindowImpl - ToolWindow icons should be 13x13. Please fix ToolWindow (ID:  PsiViewer) or icon jar:file:/Users/…/Library/Application%20Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/193.4697.15/IntelliJ%20IDEA%202019.3%20EAP.app.plugins/psiviewer/lib/PsiViewer.jar!/images/psiToolWindow.png

Where … is my username.

That png is actually 13x13, though its DPI is 102x108.

Opened issue with JetBrains.  Please see: https://youtrack.jetbrains.com/issue/IDEA-225406

**Comment by rgoldberg** [comment]
The issue is supposedly that the icon isn't HiDPI, as per:

https://youtrack.jetbrains.com/issue/IDEA-225406#focus=streamItem-27-3758315.0-0

**Comment by hurricup** [comment]
Yes, but not sure there is a source image for an icon and I don't really want to just scale current one up. Btw, just curious, why does this warning bothers you so much?

**Comment by rgoldberg** [comment]
Nothing about this specific warning, I just don't like seeing warnings anywhere.  Either something might be going wrong, or even if all is fine, spurious warnings can make it harder to notice / find other warnings that are actually important.
